contributing_to_docs/doc_guidelines.adoc

```diff
@@ -484,6 +484,32 @@ $ lorem.sh
 ----
 ....
 
+* Use only one command per code block. For example, do not include more than one command in the same code block:
++
+Run the following commands to create templates you can modify:
++
+----
+$ oc adm create-login-template > login.html
+$ oc adm create-provider-selection-template > providers.html
+$ oc adm create-error-template > errors.html
+----
+
+* Separate a command and its related output into individual code blocks. For example:
++
+Use the `oc new-project` command to create a new project.
++
+----
+$ oc new-project my-project
+----
+
++
+Output that verifies a new project was created.
```

Comment:
I'm rethinking this line. The example could read, "The output verifies that a new project was created." Or something similar.

Reply:
^^Yes, this is better.